.dotnet/src/Custom/Common/LroHelpers.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace OpenAI;
+
+internal class LroHelpers
```

Comment:
We may be able to generalize the AssistantRunOperation implementation?  If so, such a thing could live here.